package.json

```diff
@@ -151,24 +150,29 @@
     }
   },
   "files": [
-    "src",
+    "src/index.d.ts",
+    "src/internal.d.ts",
```

Comment:
Should we leave out the internal types to avoid encouraging relying on internals that may break? Further all the internal property names are different than these types define

Reply:
if nothing currently uses them that seems good.